docs/rackhd/ubuntu_package_installation.rst

```diff
@@ -81,8 +125,8 @@ conveniently.
     sudo apt-get install on-dhcp-proxy on-http on-taskgraph
     sudo apt-get install on-tftp on-syslog
 
-Configuring RackHD
-~~~~~~~~~~~~~~~~~~~~
+**b.4** Configuring RackHD basically
```

Comment:
I don't think "basically" adds to this subtitle - perhaps the phrasing you're looking for is

"Basic RackHD configuration" or "Simple RackHD configuration"?